runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java

```diff
@@ -328,13 +347,21 @@ public void onChatMessage(ChatMessage event)
 			}
 		}
 
+		if (config.screenshotCcKick() && chatMessage.startsWith("Your request to kick"))
+		{
+			final String nameText = kickPlayerName == null ? "" : kickPlayerName + " ";
+			String fileName = "Kick " + nameText + LocalDate.now();
```

Comment:
A date is added to the screenshot by default in `ImageCapture`, don't add an extra date.